config/smtpconf.go

```diff
@@ -9,6 +9,7 @@ import (
 type SMTPConf struct {
 	SMTPAddr      string   `toml:"smtpAddr,omitempty" json:"-"`
 	SMTPPort      string   `toml:"smtpPort,omitempty" valid:"port" json:"-"`
+	SMTPInsecure  bool     `toml:"smtpInsecure,omitempty" json:"-"`
```

Comment:
I think SMTPInsecure is confusing whether it is plain text or not. 
How about renaming it to `TLSInsecureSkipVerify`?